e2echart/e2e-chart-template.html

```diff
@@ -252,6 +259,14 @@ <h5 class="modal-title">Resource</h5>
                 return [item.locator, ` (container readiness)`, "ContainerReady"];
             }
         }
+        if (m && isKubeletlivenessCheck(item)){
```

Comment:
Is the verify complaining about the lack of a space before the bracket?

Reply:
I hadn't updated the generated bindata, fixed now and verify passed